README.md

```diff
@@ -68,7 +85,9 @@ docker build -t firecloud-orch .
 Run the container:
 ```
 docker run --name orch \
+  -p 8080:8080
```

Comment:
The expose command makes the 8080 port available from inside the container.
From the outside, it is assigned a random port. That's why you have to map
8080 to 8080 when running, so that docker doesn't assign a random port
number. Makes no sense to me.
On Sep 22, 2015 11:58 AM, "David Mohs" <notifications@github.com> wrote:

> In README.md
> <https://github.com/broadinstitute/firecloud-orchestration/pull/84#discussion_r40104865>
> :
>
> > @@ -68,7 +85,9 @@ docker build -t firecloud-orch .
> >  Run the container:
> >  ```
> >  docker run --name orch \
> > +  -p 8080:8080
>
> This surprises me. I thought the "EXPOSE 8080" in the Dockerfile took care
> of this. Is that not true? If not, then I don't even know what the EXPOSE
> line is supposed to do. So much to learn.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/broadinstitute/firecloud-orchestration/pull/84/files#r40104865>
> .
>